vllm/model_executor/models/qwen2_5_vl.py

```diff
@@ -462,6 +465,10 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return x
 
 
+@set_model_tag("Qwen2_5_VisionPatchMerger")
+@support_torch_compile(dynamic_arg_dims={
+    "x": 0,
```

Comment:
qq: what happens to other dimensions? does dynamism on other dimensions trigger a recompile?

Reply:
Yes dynamism on the other dimensions would trigger recompile here since they aren't explicitly marked